content/reference/user/login/index.md

```diff
@@ -6,20 +6,35 @@ lastmod: 2021-03-10T09:29:04-05:00
 weight: 10010
 ---
 
-In order to log in, you need an account with the same username as your GitHub account.
+To log in, you need a GitHub account.
 
-If you're the root administrator, we've already created your account.
-
-Otherwise, please reach out to your root administrator to [create your account]({{< ref "../../admin/users#managing-users" >}}).
+If you want to connect with an existing organization, please reach out to your root administrator to [join an organization]({{< ref "../../admin/users#managing-users" >}}).
 
 {{< img src="login-github.png" alt="Login button" >}}
 
-Pressing the **Signup with GitHub** button will redirect you to GitHub, where you will be asked to authorize the application.
+Pressing the **Continue with GitHub** button will redirect you to GitHub, where you will be asked to authorize the application.
 
-You need to **authorize** it. We only ask access to your profile.
+You need to **authorize** it. We only ask access to your profile and email address.
 
 {{< img src="login-github-oauth.png" alt="OAuth GitHub page" >}}
 
 {{< alert danger >}}
 The GitHub authorization page is opened in a new tab. If no tab opened, make sure that your browser didn't block it.
 {{< /alert >}}
+
+You need to fill the Signup form to create an account on our application.
+
+{{< img src="signup-gatling.png" alt="OAuth GitHub page" >}}
+
+Pressing the **Create account** button to register you on our application. 
+
+If a root administrator added you to his organization, you're logged in.
```

Comment:
I don't think we show the signup form to a user if their account was created be a system admin, do we? So this isn't quite correct (the user won't even see a "create account" button in this case).

Reply:
system admin, not root administrator

Reply:
Done